Source/RxCocoa/UIViewPropertyAnimator+Rx.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  UIViewPropertyAnimator+Rx.swift
+//  RxSwiftExt
+//
+//  Created by Wittemberg, Thibault on 29/03/18.
+//  Copyright © 2017 RxSwift Community. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import RxSwift
+import RxCocoa
+
+@available(iOSApplicationExtension 10.0, *)
+extension Reactive where Base == UIViewPropertyAnimator {
+
+    /// Completable that when subscribed to, starts the animation
+    /// and completes once the animation is ended
+    public var animate: Completable {
+        return Completable.create(subscribe: { (completable) -> Disposable in
+
+            self.base.addCompletion({ (position) in
```

Comment:
Same here about trailing closure syntax, and I'd also use a guard: 

```swift
base.addCompletion { position in 
    guard position == .end else { return }
    completable(.completed)
}
```

Reply:
done.